index.js

```diff
@@ -109,6 +109,30 @@ class DHT extends EventEmitter {
     return socket ? socket.address() : null
   }
 
+  localAddress () {
+    if (!this.io.serverSocket) return null
```

Comment:
This line and another line are the only differences between the methods found on `hyperdht`